common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -216,7 +216,10 @@
                     @standfirst(item)
                     @mediaMeta(item)
                 } else {
-                    <div class="fc-item__standfirst-wrapper">
+                    <div class="@RenderClasses(Map(
```

Comment:
@regiskuckaertz I've updated this PR to support timestamps on all cards. Now if item.timeStampDisplay.isEmpty is `false` we add the class `fc-item__standfirst-wrapper--timestamp`. This modifies the `standfirst-wrapper` styles to accommodate a timestamp in the `aside`.

Reply:
Great!